src/index.html

```diff
@@ -1,17 +1,174 @@
+        <label class="information label">
+        Full date of birth:
+          <input
+            type="date"
+            name="date"
+            required
+          >
+        </label>
+        <label class="information label">
+        I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="remeber"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            class="email"
+            required
+          >
+        </label>
+        <label class="information label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="16"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="radio"
+              required
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="radio"
+              required
+            >
+            No
+          </label>
+        </div>
+        <div class="information">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              required
+            >
+          </label>
+        </div>
+        <div class="information">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+              required
+            >
+          </label>
+        </div>
+        <div class="information">
+          <label>
+            What are your favorite brand of cars?
+            <select name="cars" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="information">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              name="range"
+              required
+              min="0"
+              max="100"
+              value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <div >
+          <label>
+            Comments:
+            <textarea
+              name="Comments"
+              cols="20"
+              rows="2"
+              required
+            >
+            </textarea>
```

Comment:
```suggestion
            ></textarea>
```
to avoid empty space when you click on textarea